app/templates/message_template.hbs

```diff
@@ -22,7 +22,7 @@
               {{display-message-time message.createdAt}}
             </div>
 
-            {{message.body}}
+            {{{message.body}}}
```

Comment:
Hi @steveyken , Just to apply styles(or tags) in the text.(like "html_safe" in rails)
 http://handlebarsjs.com/#html-escaping